frontend/components/Link.tsx

```diff
@@ -21,12 +21,26 @@ function Link({
   style?: React.CSSProperties;
   color?: string,
 }) {
+  if (href === undefined) {
```

Comment:
I'd prefer something like the code below to avoid duplication.
```tsx
const Wrapper = href ? <NextLink> : <>;
return (
  <Wrapper>
    <MuiLink /*props...*/ >
      {children}
    </MuiLink>
  </Wrapper>
)
```

Reply:
Maybe it's supposed to be `const Wrapper = href ? NextLink : React.Fragment;` instead, I always get them mixed up.

Reply:
That looks good